docs/narr/startup.rst

```diff
@@ -132,7 +132,7 @@ Here's a high-level time-ordered overview of what happens when you press
 #. ``pserve`` starts the WSGI *server* defined within the ``[server:main]``
    section.  In our case, this is the Waitress server (``use =
    egg:waitress#main``), and it will listen on all interfaces on port 6543
-   for both IPv4 and IPv6 (``listen = 127.0.0.1:6543 [::1]:6543``). The server
+   for both IPv4 and IPv6 (``listen = localhost:6543``). The server
```

Comment:
This may not be the case on certain Linux distributions that have a seperate ipv6-localhost type entry in /etc/hosts and localhost is not considered IPv6. 

Ubuntu 12.04/14.04 for example:
```
127.0.0.1	localhost

# The following lines are desirable for IPv6 capable hosts
::1     ip6-localhost ip6-loopback

```

Other systems also sometimes have IPv6 as:

```
::1 localhost6 localhost6.localdomain
```

Changing this also means that the user needs to make sure they have the appropriate entries set up, and a functioning resolver. I am a -0 on this change as a whole.